src/OpenTelemetry.Exporter.Geneva/README.md

```diff
@@ -270,23 +270,72 @@ For example:
 
 ##### OtlpProtobufEncoding
 
-On Windows set `PrivatePreviewEnableOtlpProtobufEncoding=true` on the
-`ConnectionString` to opt-in to the experimental feature for changing the
-underlying serialization format to binary protobuf following the schema defined
-in [OTLP
+An experimental feature flag is available to opt-into changing the underlying
+serialization format to binary protobuf following the schema defined in [OTLP
 specification](https://github.com/open-telemetry/opentelemetry-proto/blob/v1.1.0/opentelemetry/proto/metrics/v1/metrics.proto).
 
+When using OTLP format `Account` and `Namespace` are **NOT** required to be set
```

Comment:
```suggestion
When using OTLP protobuf encoding, `Account` and `Namespace` are **NOT** required to be set
```

I suggest to use OTLP protobuf encoding and not just OTLP to avoid confusing with OTLP Exporter... Other than encoding metrics using OTLP protobuf, there is no connection to OTLP here!